src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+
+      {
+        todos.map(todo => (
+          <TodoItem key={todo.id} todo={todo} />
+        ))
+      }
+
+      {/* This todo is being edited */}
+      {false && (
```

Comment:
Replace the hardcoded `false` with a state variable to toggle the 'editing' state of a todo.

Reply:
Done